lib/foreman_discovery/host_converter.rb

```diff
@@ -0,0 +1,26 @@
+class ForemanDiscovery::HostConverter
```

Comment:
Frankly I don't see a reason why not putting it into app/models, but app/services works for me as well. Also I don't think staypuft will adopt this change anytime soon.